dragonfly/lib/refinery/dragonfly/dragonfly.rb

```diff
@@ -32,7 +32,7 @@ def configure!(extension)
 
           #  These options require a name and block
           define_url extension.dragonfly_define_url if extension.dragonfly_define_url.present?
-          before_serve extension.dragonfly_before_serve if extension.dragonfly_before_serve.present?
+          before_serve(&extension.dragonfly_before_serve) if extension.dragonfly_before_serve.present?
```

Comment:
makes sense, otherwise it'd get immediately executed

Reply:
Since you're passing in a block, it would be a `Proc` object at this point. So it would only execute if you invoked the `call` method on it. The `&` is converting the `Proc` object back into a block since the `before_serve` method is expecting a block.

I'm pretty sure that is the case, let me know if I mistook something.